ksp_plugin_adapter/main_window.cs

```diff
@@ -219,23 +219,23 @@ internal class MainWindow : VesselSupervisedWindowRenderer {
           if (UnityEngine.GUILayout.Button(
                   L10N.CacheFormat("#Principia_MainWindow_Declutter"),
                   GUILayoutWidth(5))) {
-            show_only_pinned_markers = true;
-            show_only_pinned_celestials = true;
-            hide_equipotentials = true;
+            show_unpinned_markers = false;
+            show_unpinned_celestials = false;
+            show_equipotentials = false;
```

Comment:
I am noticing that this state is not persisted.  It should be, it's not nice for the user to have to unclutter each time.  Please update `Save` and `Load` accordingly.

Reply:
It should not be; the point of decluttering is that it is highly situational, and not very fine grained.
We don’t want people who pressed the « get the noodles out of my face » button once when they were looking at a long flight plan, to never see a single equipotential.

Reply:
Ack.